app/Http/Controllers/HinarioListController.php

```diff
@@ -21,29 +22,32 @@ public function __construct(PersonService $personService)
 
     public function individual()
     {
-        $hinarios = Hinario::where('type_id', HinarioTypes::INDIVIDUAL)
-            ->with('receivedBy')
-            ->get();
-
-        $people = [];
-        $tableOfContents = [];
-        foreach($hinarios as $hinario) {
-            $people[$hinario->receivedBy->list_order][$hinario->receivedBy->display_name] = $hinario->receivedBy;
-            $tableOfContents[$hinario->receivedBy->display_name] = $hinario->receivedBy;
-        }
-
-        $listOrders = array_keys($people);
-        rsort($listOrders);
-
-        $peopleList = [];
-        foreach ($listOrders as $listOrder) {
-            ksort($people[$listOrder]);
-            $peopleList = array_merge($peopleList, $people[$listOrder]);
-        }
-
-        ksort($tableOfContents);
+        $persons = Person::has('hinarios')->with('translations','images','personImages','hinarios')->orderBy('display_name')->get();
+        $personsByListOrder = $persons->sortByDesc('list_order');
+        // orderBy('list_order','desc')
```

Comment:
Should probably just get rid of the commented out stuff unless you think you'll need it.